### PR TITLE
Mark the SeleniumDriver as deprecated in favor of Selenium2

### PR DESCRIPTION
**Pull request by stof**

### CODE REVIEW

File: drivers/selenium.rst [hunk 1]
```diff
@@ -5,6 +5,13 @@ SeleniumDriver provides a bridge for the famous `Selenium`_ tool. If you
 need legacy Selenium, you can use it right out of the box in your Mink test
 suites.
 
+.. caution::
+
+    The SeleniumRC protocol used by this driver is deprecated and does not
+    support all Mink features. For this reason, the SeleniumDriver is deprecated
+    in favor of the :doc:`/drivers/selenium2`, which is based on the new
```
Comment (aik099) [inline drivers/selenium.rst]: I'm not sure if leading `/` in link to other document is needed.

**Comment by aik099** [comment]
Maybe we should also update `README.md` of MinkSeleniumDriver repo?

**Comment by stof** [comment]
@aik099 an update of all driver readmes is planned to link to the online doc. This will be part of it.

File: drivers/selenium.rst (hunk 1 above)
Comment (stof) [inline drivers/selenium.rst]: Well, we have 2 choices: either absolute references (which need the leading slash) or relative references (which would be ``:doc:`selenium2` ``). I'm using absolute references everywhere currently (easier when moving a file to a different location as all references inside it don't need to be changed, and searching for references to a given file is also easier)

Comment (aik099) [inline drivers/selenium.rst]: OK.

**Pull request merged**